.changeset/new-cows-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
```

Comment:
I think this is a new non-breaking feature, so I would consider it a minor change